src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,7 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    public String getFeedbackToUser() {
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.